packages/zoe/src/cleanProposal.js

```diff
@@ -100,62 +157,27 @@ export const cleanKeywords = keywordRecord => {
  * `{ waived: null }` `{ onDemand: null }` `{ afterDeadline: { timer
  * :Timer, deadline :bigint } }
  *
- * @param {(brand: Brand) => DeprecatedAmountMath} getAmountMath
  * @param {Proposal} proposal
  * @returns {ProposalRecord}
  */
-export const cleanProposal = (getAmountMath, proposal) => {
+export const cleanProposal = proposal => {
   const rootKeysAllowed = ['want', 'give', 'exit'];
```

Comment:
No reason to evaluate the constant array expression to a new array every time the function is called. I just pull such constant data out to the module top level. Same comment applies to the `firstCapASCII` regexp above. Not a big deal, but a good habit to develop.

```suggestion
const rootKeysAllowed = harden(['want', 'give', 'exit']);

...comment goes here...
export const cleanProposal = proposal => {
```

Reply:
Done!